config/settings.local.yml.template

```diff
@@ -1,5 +1,8 @@
 # This will override what is in settings.yml, which is committed to version control.
 
+devise:
+  secret_key:
```

Comment:
Actually, what do you think of adding something to `settings/development.yml` like I did for `test.yml`. That way it's one less step you need to deal with when doing initial setup, but something you would still need to configure when doing a stage/prod deploy?